lib/jsonapi/view.ex

```diff
@@ -59,6 +59,17 @@ defmodule JSONAPI.View do
   If you always want to include a relationship. First make sure its always preloaded
   and then use the `[user: {UserView, :include}]` syntax in your `includes` function. This tells
   the serializer to *always* include if its loaded.
+
+  There are several options available through configuration. These include switching from underscores
```

Comment:
IMO inline documentation should be more of just that: documentation.  A code example does not really inform what these do.  I think if we intend to document options in the code, we should follow a similar pattern to Elixir and other libraries: https://github.com/elixir-lang/elixir/blob/master/lib/elixir/lib/string.ex#L297